packages/audiofileplayer/android/src/main/java/com/google/flutter/plugins/audiofileplayer/AudiofileplayerService.java

```diff
@@ -62,8 +62,13 @@ public void onCreate() {
     super.onCreate();
     Log.i(TAG, "onCreate");
     instance = this;
+    PendingIntent pendingItent = PendingIntent.getBroadcast(
+            getApplicationContext(),
+            0, new Intent(Intent.ACTION_MEDIA_BUTTON),
+            PendingIntent.FLAG_IMMUTABLE
+    );
 
-    mediaSession = new MediaSessionCompat(this, TAG);
+    mediaSession = new MediaSessionCompat(this, TAG,null,pendingItent);
```

Comment:
Looks like this intent is only used up to KitKat:
https://developer.android.com/reference/android/support/v4/media/session/MediaSessionCompat#MediaSessionCompat(android.content.Context,%20java.lang.String,%20android.content.ComponentName,%20android.app.PendingIntent)

1) Does this library support KitKat or farther back?
2) If so, add a comment highlighting that, so that if this library ever drops support for KitKat and previous, to remove this pendingIntent